### ISSUE TITLE
[Alias] Finish alias grouping

### ISSUE THREAD

**Issue by ckx** [Clam-/pyBurlyBot#70]
Alias groups that allow you to add users to a collective group.

For example, the "GTA" alias might return everyone who plays GTA V. Would make sending tells about specific game playtimes and other stuff like that easier.

**Comment by gkistler** [comment]
Also, provide a way to map an alias to a group, e.g.

```
.alias ~group onesquare
```

->

```
onesquare fire_barrell horrible fgt worstmusicever fgtcakes biggershitlord gayfin greedfygriff virus_2.0_beta animufag lactosebro mumanddaduser
``

Basically get start  on ideas of getting "negative jokes in brocircle -:> giba one user -> gib all jokes with same ideas.  Just think about it and try to get .group working.  Will collab more soon if mambs

**Comment by Clam-** [comment]
Currently refactoring alias groups.
I suppose group aliases would be neat thing... probably won't be in my first refactoring to fully finalise group stuffs, but we'll see.

**Comment by Clam-** [comment]
Implemented in bc66344
(could probably be fine tuned a little, e.g. adding multiple users in one .group command, watev)
